torch/nn/_functions/thnn/batchnorm_double_backwards.py

```diff
@@ -0,0 +1,97 @@
+from torch.autograd.variable import Variable
+from functools import reduce
+from operator import mul
+
+
+def sum_exclude_dim1(to_sum, keepdim=True):
+    to_sum = to_sum.sum(dim=0, keepdim=True)
+    dim = 2
```

Comment:
no, good catch.  This code also doesn't respect the keepdim parameter (and wouldn't use it correctly if it did).  Let me rewrite.